test/test_turbulence_closures.jl

```diff
@@ -178,9 +178,9 @@ function compute_closure_specific_diffusive_cfl(closure)
     model = NonhydrostaticModel(; grid, closure, buoyancy=BuoyancyTracer(), tracers=:b)
     dcfl = DiffusiveCFL(0.1)
     @test dcfl(model) isa Number
-    @test diffusive_flux_x(1,1,1, grid, model.closure, model.diffusivity_fields, ::Val{1}, model.tracers.b, model.clock, fields(model), model.buoyancy) == 0
-    @test diffusive_flux_y(1,1,1, grid, model.closure, model.diffusivity_fields, ::Val{1}, model.tracers.b, model.clock, fields(model), model.buoyancy) == 0
-    @test diffusive_flux_z(1,1,1, grid, model.closure, model.diffusivity_fields, ::Val{1}, model.tracers.b, model.clock, fields(model), model.buoyancy) == 0
+    @test diffusive_flux_x(1,1,1, grid, model.closure, model.diffusivity_fields, Val(1), model.tracers.b, model.clock, fields(model), model.buoyancy) == 0
+    @test diffusive_flux_y(1,1,1, grid, model.closure, model.diffusivity_fields, Val(1), model.tracers.b, model.clock, fields(model), model.buoyancy) == 0
+    @test diffusive_flux_z(1,1,1, grid, model.closure, model.diffusivity_fields, Val(1), model.tracers.b, model.clock, fields(model), model.buoyancy) == 0
```

Comment:
Spacing // formatting is wrong

Reply:
@glwagner can you be specific? I don't see anything wrong with the formatting in these lines

Reply:
I see. I didn't know this was something that was enforced. Since it was just three ones in sequence I thought I'd save some spaces, but I'll change it.

Reply:
We always put spaces after commas (like English). Generally our perspective is that formatting is crucial.

See:

* https://clima.github.io/OceananigansDocumentation/stable/contributing/#But-I-want-to-*code*!
* https://github.com/jrevels/YASGuide#linealignmentspacing-guidelines